episodes/chapter8.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Work on your own project"
+teaching: 10
+exercises: 1
+---
+
+:::::::::::::::::::::::::::::::::::::: questions
+
+- How can you use vantage6 in your own project?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Start working on your own research project
+
+OR
+
+- Complete some advanced challenges
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+# Working on your own project
+
+In this chapter, we will give you the opportunity to work on your own project. Feel free
+to ask questions and discuss your project with the instructors. We are here to help you!
+
+If you prefer, you can also complete some advanced challenges that we have prepared for
+you.
+
+# Advanced challenges
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge: Add privacy filters to your algorithm
+
+In the [previous lesson](./algorithm_development.md), you have created a simple
+algorithm. Now, you can add privacy filters to your algorithm.
+
+- Create an algorithm that does not return the results unless there are more than 10
+  data points.
+- Make this value configurable by the node administrator. Hint: look up the node
+  configuration options in the [documentation](https://docs.vantage6.ai) to provide
+  environment variables to your algorithm.
+- What else could you do to protect the privacy of the data?
+
```

Comment:
Maybe also add an example of where think it is done properly (crosstabs) and maybe also a reference to the exceptions? https://docs.vantage6.ai/en/main/function-docs/_autosummary/vantage6.algorithm.tools.exceptions.html

Reply:
Added crosstab as a hint, and using the errors as part of the challenge